core/unix/signal_private.h

```diff
@@ -191,7 +191,7 @@ typedef _STRUCT_UCONTEXT /* == __darwin_ucontext */ kernel_ucontext_t;
  * (these are from /usr/src/linux/arch/i386/kernel/signal.c for kernel 2.4.17)
```

Comment:
When I first looked at this CL I thought it was ignoring all #include lines, yet it does change some of them.  How does it identify the ones it is changing?  E.g., in fragment.h for `# include "hashtablex.h"`.